packages/select/src/common/listItemsProps.ts

```diff
@@ -107,11 +108,20 @@ export interface IListItemsProps<T> extends IProps {
     noResults?: React.ReactNode;
 
     /**
-     * Invoked when user interaction should change the active item: arrow keys move it up/down
-     * in the list, selecting an item makes it active, and changing the query may reset it to
-     * the first item in the list if it no longer matches the filter.
+     * Invoked when user interaction should change the active item: arrow keys
+     * move it up/down in the list, selecting an item makes it active, and
+     * changing the query may reset it to the first item in the list if it no
+     * longer matches the filter.
+     *
+     * If the "Create Item" option is displayed and currently active, then
+     * `isCreateNewItem` will be `true` and `activeItem` will be `null`. In this
+     * case, you should provide a valid "Create Item" object to the `activeItem`
+     * _prop_ in order for the "Create Item" option to appear as active.
+     *
+     * __Note:__ You can instantiate a "Create Item" object using the
+     * `getCreateNewItem()` utility exported from this package.
      */
-    onActiveItemChange?: (activeItem: T | null) => void;
+    onActiveItemChange?: (activeItem: T | null, isCreateNewItem: boolean) => void;
```

Comment:
is `(activeItem: T | null) => void` (the old typedef) assignable to this new type? if not, it's a break, and we can mitigate that by making the second param optional.

Reply:
@adidahiya Yes, it's assignable.

You could assign an incorrect second-parameter type, and TS would complain:

![image](https://user-images.githubusercontent.com/443450/53514050-a9764680-3a7b-11e9-968d-15890f029e0a.png)

But simply omitting the parameter is acceptable:

![image](https://user-images.githubusercontent.com/443450/53514006-906d9580-3a7b-11e9-813c-b5fdbf3ac133.png)

Reply:
Another example with component usage. Works great. 👍 

![image](https://user-images.githubusercontent.com/443450/53514804-674e0480-3a7d-11e9-8f52-1085bebf6e1a.png)